docs/Configuration.md

```diff
@@ -1144,6 +1144,37 @@ Default: `'prettier'`
 
 Sets the path to the [`prettier`](https://prettier.io/) node module used to update inline snapshots.
 
+Note that Prettier version 3 is not supported. You can either pass `prettierPath: null` in your config to disable using prettier if you don't need it, or use v2 of Prettier solely for Jest.
```

Comment:
I've forgotten to roll 29.6 docs - will do so after landing this

Reply:
What about having this block in some `:::info` admonition?

Reply:
Or perhaps like this:

```md
<details>
  <summary>Prettier version 3 is not supported!</summary>
  Etc...
</details>
```

Reply:
ooh, `details` look great! didn't know docusaurus formatted it

Reply:
https://deploy-preview-14367--jestjs.netlify.app/docs/next/configuration#prettierpath-string